### DISCUSSION TITLE
Display products in homepage

### DISCUSSION THREAD

**Discussion by IncredibleAshish316** [saleor/saleor#9546] [General]
I am trying to categories product in homepage of my react storefront but I haven't been able to solve. Can anyone tell what homepagequeryblock do ??

**Comment by krzysztofwolski** [comment]
It renders content of the Menu object as sections on the homepage:
- collections/categories are rendered as list of products 
- the content of the Pages is fully displayed

**Reply by IncredibleAshish316** [reply]
Yes, that what I am expecting but I am not getting any products on homepage only on search
